scripts/apps/authoring/authoring/index.ts

```diff
@@ -140,6 +140,27 @@ angular.module('superdesk.apps.authoring', [
                     return authoring.itemActions(item).edit;
                 }],
             })
+            .activity('edit.media.metadata', {
+                label: gettext('Edit Media Metadata'),
+                priority: 3,
+                icon: 'edit-line',
+                keyboardShortcut: 'ctrl+alt+i',
```

Comment:
what about `ctrl+alt+m` given that it's about media

Reply:
yes, right. But I there is also a `metasearch` so I thought it could be used for that. What do you think?

Reply:
should be fine, that's ansa specific, so we can change it for them if needed, but afaik there is no shortcut now for metasearch

Reply:
my bad, seems like there should be a shortcut (SDANSA-233), so not sure what to do now

Reply:
done

Reply:
@petrjasek I discussed it with Karel, and he says that keep `ctrl+alt+m`  for `edit media metadata`.